x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/chat/chat.spec.ts

```diff
@@ -68,6 +70,46 @@ export default function ApiTest({ getService }: DeploymentAgnosticFtrProviderCon
       expect(status).to.be(404);
     });
 
+    it('returns a 200 if the connector exists', async () => {
+      void proxy
+        .intercept('conversation', () => true, 'Hello from LLM Proxy')
+        .completeAfterIntercept();
+      const { status } = await observabilityAIAssistantAPIClient.editor({
+        endpoint: 'POST /internal/observability_ai_assistant/chat',
+        params: {
+          body: {
+            name: 'my_api_call',
+            systemMessage: '',
+            messages,
+            connectorId,
+            functions: [],
+            scopes: ['all'],
+          },
+        },
+      });
+      expect(status).to.be(200);
```

Comment:
Ensure that all intercepts have been called.

```suggestion
      await proxy.waitForAllInterceptorsSettled();
      expect(status).to.be(200);
```

Btw. wdyt about renaming `waitForAllInterceptorsSettled()` to `allInterceptorsToHaveBeenCalled()` ?

Reply:
`allInterceptorsToHaveBeenCalled()` sounds a clearer and more descriptive name. I’m good doing the change in this PR, do you think should I do it in this PR or open a issue and create a new PR?

Reply:
Up to you. I'm fine with both